+ python3 duplicate_finder.py --path /share/lasallelab/Oran/
Traceback (most recent call last):
  File "duplicate_finder.py", line 38, in <module>
    files_and_sizes[os.path.join(path, name)] = os.path.getsize(os.path.join(path, name))
  File "/software/anaconda3/4.8.3/lssc0-linux/lib/python3.8/genericpath.py", line 50, in getsize
    return os.stat(filename).st_size
FileNotFoundError: [Errno 2] No such file or directory: '/share/lasallelab/Oran/dovetail/luhmes/merged/oj/lib/libstdc++.so.6.0.21'
